lib/fetch.js

```diff
@@ -55,6 +54,32 @@ export function mapAndSanitate(item, type) {
   };
 }
 
+export function sanitize(key, type, defaultValue, json) {
```

Comment:
Wont this also be changed when we use the library you are working on @jameshopkins ?